### PR TITLE
Ns 6

### PR DESCRIPTION
**Pull request by Poostaq**

### CODE REVIEW

**Review by Malynna — CHANGES_REQUESTED** [review]
change specialization in files and files names

File: game/MainScreen.gd [hunk 1]
```diff
@@ -14,7 +14,8 @@ export (NodePath) onready var ethnicity_element = get_node(ethnicity_element) as
 export (NodePath) onready var profession_element = get_node(profession_element) as Control
 export (NodePath) onready var attributes_element = get_node(attributes_element) as Control
 export (NodePath) onready var specialisation_element = get_node(specialisation_element) as Control
```
[the remark this answers is not in the document]
Comment (Poostaq) [reply game/MainScreen.gd]: fixed

File: game/MainScreen.tscn [hunk 2]
```diff
@@ -44,7 +45,8 @@ ethnicity_element = NodePath("CenterContainer/VBoxContainer/StepContainer/Ethnic
 profession_element = NodePath("CenterContainer/VBoxContainer/StepContainer/ProfessionStep")
 attributes_element = NodePath("CenterContainer/VBoxContainer/StepContainer/AttributesStep")
 specialisation_element = NodePath("CenterContainer/VBoxContainer/StepContainer/SpecialisationStep")
```
[the remark this answers is not in the document]
Comment (Poostaq) [reply game/MainScreen.tscn]: fixed

File: game/SpecialisationPage/SpecialisationStep.gd [hunk 3]
```diff
@@ -52,6 +52,7 @@ func load_step() -> void:
 	current_specialization = db.read_data_for_specialisation(specialisation_id)
 	_load_specialisation(current_specialization)
```
[the remark this answers is not in the document]
Comment (Poostaq) [reply game/SpecialisationPage/SpecialisationStep.gd]: fixed

**Review by Malynna — APPROVED** [review]

**Pull request merged**